utils/utils.js

```diff
@@ -187,8 +187,8 @@ const utils = {
    */
   escapeText: escapeMarkdown,
   /**
-   * Returns a MessageEmbed with basic values preset, such as color and timestamp.
    * @param {{author?: Discord.GuildMember|Discord.User|Discord.APIEmbedAuthor|Discord.EmbedAuthorData|null} & Omit<(Discord.Embed | Discord.APIEmbed | Discord.EmbedData), "author">} [data] The data object to pass to the MessageEmbed constructor.
+   * @returns {Discord.MessageEmbed} a Discord.MessageEmbed with basic values preset, such as color and timestamp.
```

Comment:
Is this a needed change? The TS server (which powers intellisense) has provided MessageEmbed as the return type without this in the past.

Reply:
I probably have something configured wrong because the TS checker really, really hates me right now, and shows things as deadly errors even though they evidently are not, and its just seemingly having a panic attack or smth. will revert change.